gemviz/mainwindow.py

```diff
@@ -3,7 +3,11 @@
 import __init__
 import utils
 from app_settings import settings
+from PyQt5.QtCore import QUrl
 
+LOCALHOST_URL = "http://localhost:8000"
+TESTING_URL="http://otz.xray.aps.anl.gov:8000"
+TILED_SERVER_SETTINGS_KEY = "tiled_server"
```

Comment:
Why did this symbol move between modules?

Reply:
Issue #149 would help here.

Reply:
> Why did this symbol move between modules?

I added it here because I needed to use it to populate the server `QComboBox`. Is that a problem?

Reply:
I think it should be imported from the tiledserver module (keeping similar things together, such as for #149).